redaxo/src/core/lib/util/i18n.php

```diff
@@ -101,59 +101,28 @@ public static function rawMsg($key)
     }
 
     /**
-     * Returns the translation for the given key.
+     * Returns the translation htmlspecialchared for the given key and locale.
      *
-     * @param string $key
-     * @param bool   $htmlspecialchars
-     * @param array  $args
+     * @param string $key    A Language-Key
+     * @param string $locale A Locale
      *
-     * @return mixed
+     * @return string Translation for the key
      */
-    private static function getMsg($key, $htmlspecialchars, array $args)
+    public static function msgInLocale($key, $locale)
```

Comment:
ggf. sollte man hier `$htmlspecialchars` auch via parameter zulassen, damit man `msgInLocale()` für alle use-cases nutzen kann. da man messages in nicht-default locale eher selten brauch, würde ich dafür - nicht wie bei `msg()` und `rawMsg()` - eine separate methode anbieten wollen.

feedack dazu?

Reply:
Bin mir unsicher. Tendiere aber leicht dazu, es lieber einheitlich zu machen, und dann auch eine raw-Methode einzuführen.

Reply:
Aktuell brauchen wir keine raw-methode, daher dann erstmla so lassen?

Reply:
Ich glaube, ich würde sie trotzdem direkt mit einführen.

Reply:
ergänz